telluric/georaster.py

```diff
@@ -738,10 +738,21 @@ def __contains__(self, geometry):
         """
         return self.footprint().contains(geometry)
 
-    def astype(self, dst_type, stretch=False):
+    def astype(self, dst_type, in_range='dtype', out_range='dtype', clip_negative=False, stretch=False):
         """ Returns copy of the raster, converted to desired type
-        Supported types: uint8, uint16, uint32, int8, int16, int32
-        For integer types, pixel values are within min/max range of the type.
+        Supported types: uint8, uint16, uint32, int8, int16, int32, float16, float32, float64
+
+        :param dst_type: desired type
+        :param in_range: str or 2-tuple, default 'dtype':
+            'image': use image min/max as the intensity range,
+            'dtype': use min/max of the image's dtype as the intensity range,
+            2-tuple: use explicit min/max intensities
```

Comment:
can I specify in range only one part for example (1, max) so it will set the range from 1 - image.max? or (min, 100)?

Reply:
yes, it might be nice if I put `'min'` it will replace it with `min(some_raster.min())`